mindsdb/integrations/handlers/vertex_handler/vertex_handler.py

```diff
@@ -0,0 +1,55 @@
+from mindsdb.integrations.libs.base import BaseMLEngine
+from mindsdb.integrations.handlers.vertex_handler.vertex_client import VertexClient
+from mindsdb.utilities import log
+import pandas as pd
+
+
+class VertexHandler(BaseMLEngine):
+    """Handler for the Vertex Google AI cloud API"""
+
+    name = "Vertex"
+
+    def create(self, target, df, args={}):
+        """Logs in to Vertex and deploy a pre-trained model to an endpoint.
+
+        If the endpoint already exists for the model, we do nothing.
+
+        If the endpoint does not exist, we create it and deploy the model to it.
+        The runtime for this is long, it took 15 minutes for a small model.
+        """
+        assert "using" in args, "Must provide USING arguments for this handler"
+        model_name = args["using"]["model_name"]
+        service_key_path = args["using"]["service_key_path"]
```

Comment:
I think I will update it later (in different PR), will use here
- url to service file
- and/or service file content
And will enable use it via create ml engine. 
It is all need to use it on our cloud